addons/account_sepa/account_sepa_credit_transfer.py

```diff
@@ -0,0 +1,380 @@
+# -*- coding: utf-8 -*-
+
+import re
+import time
+import random
+import base64
+
+from unidecode import unidecode
+
+from openerp import models, fields, api, _
+from openerp.tools import float_round
+from openerp.exceptions import UserError, ValidationError
+
+''' TODO
+    - Improve res.partner.bank
+    - Check base_iban is reliable (eg. get_bban_from_iban contains a TODO)
```

Comment:
Unfortunately no, the financial institutions weren't capable to come up with a real standard so they made a standard filled with special cases and conditions. Search 'generic' in this document to get an idea : https://www.febelfin.be/sites/default/files/files/Standard-CreditTransfer-v2-EN_1.pdf